azure-pipelines.yml

```
@@ -251,20 +251,16 @@ stages:
           & dotnet build $testProjFileName -c $(BuildConfiguration)
           Test-ExitCode "ERROR: UnitTest project build FAILED."
 
-          Write-Host "List Sonar*.dll files"
-          Get-ChildItem -Recurse .\tests\SonarAnalyzer.UnitTest\bin\${BuildConfiguration}\ -Filter Sonar*.dll
+          Write-Host "Register OpenCover profilers. This step needs to be done manually on CI agents. See https://github.com/OpenCover/opencover/raw/master/main/OpenCover.Documentation/Usage.pdf"
```

It seems to me that only the beginning is a message. And the rest is a comment that's actionable only in the YML file, right?
```suggestion
          Write-Host "Register OpenCover profilers."
          #This step needs to be done manually on CI agents. See https://github.com/OpenCover/opencover/raw/master/main/OpenCover.Documentation/Usage.pdf"
```